.github/scripts/instr-count-benchmark/create-nightly-env.sh

```diff
@@ -7,9 +7,16 @@ set -a;
 source ${BASEDIR}/config.env
 set +a;
 
+mkdir -p ${INSTRUCTION_COUNT_ROOT}
+
 conda create -y -q --name ${CONDA_ENV_NAME} python=${PYTHON_VERSION}
 . activate ${CONDA_ENV_NAME}
 
+# We need Valgrind to collect instructions.
+conda install -y valgrind -c conda-forge
+
 # Install PyTorch nightly from pip
 pip install --pre torch \
```

Comment:
Sometimes pytorch nightly will fail to build, and we should cancel CI. Please add a check similar to here: https://github.com/pytorch/benchmark/blob/0.1/.github/scripts/run-nightly-nodocker.sh#L33
```suggestion
# Check if nightly builds are available
NIGHTLIES=$(python torchbenchmark/util/torch_nightly.py --packages torch)
# If failed, the script will generate empty result
if [ -z $NIGHTLIES ]; then
    echo "Torch, torchvision, or torchtext nightly build failed. Cancel the workflow."
    exit 1
fi
pip install --pre torch \
```

Reply:
Thanks for the tip.